test/helper.rb

```diff
@@ -125,6 +157,31 @@ def skip_unless(eng, bt: caller)
     end
     skip skip_msg, bt if skip_msg
   end
+
+  def out_io
```

Comment:
This is gonna need a comment. It looks like you're trying to make sure all ios are closed after the suite ends, right?

In that case, I think this needs assertions (not just debug output) and instead of returning against length, it would be much more descriptive if `ary` contained STDOUT/STDERR/whatever the third thing is.

Reply:
First: Correct.

The third thing (STDIN)?  Sorry.  I've never checked into whether one can fail in `Minitest.after_run`....